test/app-db/src/main/java/org/glassfish/soteria/test/ApplicationConfig.java

```diff
@@ -42,12 +42,13 @@
 import javax.enterprise.context.ApplicationScoped;
 import javax.inject.Named;
 import javax.security.enterprise.identitystore.DatabaseIdentityStoreDefinition;
+import javax.security.enterprise.identitystore.Pbkdf2PasswordHash;
 
 @DatabaseIdentityStoreDefinition(
     dataSourceLookup="${'java:global/MyDS'}", 
     callerQuery="#{'select password from caller where name = ?'}",
     groupsQuery="select group_name from caller_groups where caller_name = ?",
-    hashAlgorithm = TestPlaintextPasswordHash.class,
+    hashAlgorithm = Pbkdf2PasswordHash.class,
     hashAlgorithmParameters = {
```

Comment:
There are some standard parameters but vendors could add more to their specific implementation of the Pbkdf2 hash. I think people will expect "invalid" parameters to be ignored as it happens with JPA query hints or EntityManager properties.